test/utils/create_mock_context.js

```diff
@@ -0,0 +1,15 @@
+import spy from 'sinon/lib/sinon/spy'; // avoid babel-register-related error by importing only spy
+
+/**
+ * Returns an mock ctx object with just those properties a Feature
+ * requires.
+ *
+ * @return {Object}
+ */
+export default function createMockCtx() {
```

Comment:
Maybe should rename this to `create_mock_feature_context`, since there are a lot of "contexts" bouncing around.